Sources/Fullscreen/NotificationCenter/NotificationCenterView.swift

```diff
@@ -0,0 +1,176 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import UIKit
+
+public protocol NotificationCenterViewDelegate: AnyObject {
+    func notificationCenterView(_ view: NotificationCenterView, didSelectModelAt indexPath: IndexPath)
+    func notificationCenterView(_ view: NotificationCenterView, didSelectNotificationDetailsIn section: Int)
+    func notificationCenterView(_ view: NotificationCenterView, modelForSection section: Int) -> NotificationCenterSectionHeaderViewModel
+    func notificationCenterView(_ view: NotificationCenterView, timestampForModelAt indexPath: IndexPath) -> String?
```

Comment:
`modelForSection` and `timestampForModelAt` should maybe be in the datasource instead?

Reply:
Makes sense to move it to datasource 👍